config/smtp.rb

```diff
@@ -1,11 +1,11 @@
 SMTP_SETTINGS = {
```

Comment:
Is this required if you have `ActionMailer::Base.smtp_settings` in `config/environment.rb`?

I have a feeling that it's redundant.  So, you might want to either delete `smtp.rb` altogether or move the config from the environment config into `smtp.rb`